var/spack/repos/builtin/packages/migraphx/package.py

```diff
@@ -42,13 +44,13 @@ def url_for_version(self, version):
     depends_on('msgpack-c', type='link')
     depends_on('half@1.12.0', type='link')
 
-    for ver in ['3.5.0', '3.7.0', '3.8.0', '3.9.0', '3.10.0', '4.0.0']:
+    for ver in ['3.5.0', '3.7.0', '3.8.0', '3.9.0', '3.10.0', '4.0.0', '4.1.0']:
         depends_on('hip@' + ver, type='build', when='@' + ver)
         depends_on('rocm-cmake@' + ver, type='build', when='@' + ver)
         depends_on('llvm-amdgpu@' + ver, type='build', when='@' + ver)
         depends_on('rocblas@' + ver, type='link', when='@' + ver)
         depends_on('miopen-hip@' + ver, type='link', when='@' + ver)
-        if ver in ['3.9.0', '3.10.0', '4.0.0']:
+        if ver in ['3.9.0', '3.10.0', '4.0.0', '4.1.0']:
             depends_on('nlohmann-json')
```

Comment:
```suggestion

    depends_on('nlohmann-json@3.9.0:')
```

Reply:
@haampie  , migraphx , it still work in progress. i accidentatally committed it.. Apologies!!